app/kumactl/cmd/apply/apply_test.go

```diff
@@ -118,7 +117,8 @@ var _ = Describe("kumactl apply", func() {
 		// given
 		rootCmd.SetArgs([]string{
 			"--config-file", filepath.Join("..", "testdata", "sample-kumactl.config.yaml"),
-			"apply", "-f", filepath.Join("testdata", "apply-dataplane.yaml")},
+			"apply", "-f", filepath.Join("testdata", "apply-dataplane.yaml"),
+		},
 		)
```

Comment:
I would prefer in situations like this one, to change it to:
```suggestion
		})
```

Reply:
I don’t know how it is possible to configure gofumpt to do so . Would you ?

Reply:
Looks like if you write it as `})` it won't be reformatted. In any case I don't mind either way.